docs/spec/circuit-breaker/01_concepts.md

```diff
@@ -0,0 +1,16 @@
+# Concepts
+
+The intention of the circuit breaker is to have a contingency plan for a
+running network which maintains network liveness. This can be achieved through
```

Comment:
which desires to maintain network liveness?

Reply:
We want a contingency plan for handling possible state machine bugs which maintains the ability of governance to vote on what to do next, right? I don't think this has much to do with liveness "in general".

Reply:
Oh I think it does! - There could be an extreme case of the circuit breaker where all messages _besides_ goverance are switched off... but I don't think all circuits need to lead to this state... why not switch off a particular message type which you know is the troublemaker? no need to shut down everything all the time